packages/lit-virtualizer/test/helpers.js

```diff
@@ -12,3 +12,31 @@ export async function wait(n=0) {
     await new Promise(resolve => requestAnimationFrame(resolve));
     return new Promise(resolve => setTimeout(resolve, n));
 };
+
+/**
+ * This solution was inspired to address the issue described in the following links:
+ * https://stackoverflow.com/questions/49384120/resizeobserver-loop-limit-exceeded
+ * https://stackoverflow.com/questions/9025095/how-can-i-test-uncaught-errors-in-mocha
+ * 
+ * Before/After parameters are given to the function to ensure there are no user-error
+ * cases where teardown is forgotten/skipped.
+ */
+export function ignoreBenignErrors(before, after) {
+    ignoreWindowErrors(before, after, /^(ResizeObserver loop limit exceeded)$/);
+}
+export function ignoreWindowErrors(before, after, regexp) {
+    let onerror;
+    before(() => {
+        onerror = window.onerror;
+        window.onerror = (err) => {
+            if (regexp.test(`${err}`)) {
+                console.warn(`Ignored Error: ${err}`);
+                return false;
+            }
+            return onerror.apply(window, arguments);
+        };
+    });
+    after(() => {
+        window.onerror = onerror;
```

Comment:
If the `before` passed in is not run before `after` - maybe as developer error, should this throw? I think this could accidentally assign `undefined` to `window.onerror`.

Reply:
Hmm.  The primary reason I set the function up this way was to ensure that there wasn't an opportunity for developer error (at least of forgetting to establish a teardown hook) which would result in a teardown not being called.  I suppose a developer could always send in a function that wasn't the after function...  I don't really have a standard hook in Mocha to use in the determination of whether an expected teardown didn't happen.  If Mocha had an `aroundEach` or something I would use it:
```
aroundEach((run) { 
  const onerror = window.onerror;
  window.onerror = (err) => { ... };
  run();
  window.onerror = onerror; 
});

// or rather...
aroundEach(ignoreBenignErrors);
```

But it doesn't :shrug:

Reply:
To clarify, the ignoreBenignErrors/ignoreWindowErrors functions require passing in the setup/teardown registration functions as a means to reduce boilerplate in the devX but also to reduce the opportunity to set things up incorrectly.  That said, I suppose there are cases where these expectations might not be met for unforeseen reasons like other onerror advice methods that may get in the mix.

I also wish Mocha had an API for dealing with its uncaught exception handling.  This was the only means I found for changing it that worked in the browser.  There are other ways of effecting it when its just run by node using listeners on the process, but I tried doing this with Mocha.process in the browser and it didn't work.

Reply:
I agree. Maybe a lightweight change to check that `onerror` is not undefined before assigning it to `this.oneerror`, and throwing defensively if this is the case?

Also no action required. This is test only code.

Reply:
I missed the TypeScript conversion commit. That resolves this conversation! So awesome!